content/en/docs/Features/multi-cluster.md

```diff
@@ -3,22 +3,46 @@ title: "Multi-cluster Deployment"
 description: "Advanced Mesh Deployment and Multi-cluster support."
 ---
 
-A basic Istio mesh deployment has a single control plane with a single data plane, deployed on a single Kubernetes cluster. But Istio supports a variety of advanced 
+{{% alert color="warning" %}}
+Support for multi-cluster deployments is **experimental** and subject to change. It is not suitable for production environments.
+{{% /alert %}}
+
+A basic Istio mesh deployment has a single control plane with a single data plane, deployed on a single Kubernetes cluster. But Istio supports a variety of advanced
 [deployment models](https://istio.io/latest/docs/ops/deployment/deployment-models/). It allows a mesh to span multiple primary (control plane) and/or remote (data plane only) clusters, and can use a single or
-[multi-network](https://istio.io/latest/docs/ops/deployment/deployment-models/#multiple-networks) approach.  The only strict rule is that within a mesh service names are unique. A non-basic mesh deployment generally involves multiple clusters.  See [installation instructions](https://istio.io/docs/setup/install/multicluster/) for more detail on installing advanced mesh deployments.
+[multi-network](https://istio.io/latest/docs/ops/deployment/deployment-models/#multiple-networks) approach. The only strict rule is that within a mesh service names are unique. A non-basic mesh deployment generally involves multiple clusters. See [installation instructions](https://istio.io/docs/setup/install/multicluster/) for more detail on installing advanced mesh deployments.
+
+For multi-cluster deployments, Kiali provides a single place to manage your mesh across clusters. Support is still experimental and only primary-remote deployments are currently supported but support for more deployment models is in progress.
```

Comment:
See other comment, I think we should get away from "primary-remote" terminology.